kong/consumer_group_test.go

```diff
@@ -0,0 +1,213 @@
+package kong
+
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConsumerGroupsService(T *testing.T) {
+	RunWhenEnterprise(T, ">=2.7.0", RequiredFeatures{})
+	assert := assert.New(T)
+
+	client, err := NewTestClient(nil, nil)
+	assert.NoError(err)
```

Comment:
I know it's a theme in this repo but can we use `require` for assertions like these?

If the `err` is not `nil` in here, the test fails but continues execution and eventually panics because i.e. `client` is nil. If for whatever reason we failed to create the client (and similarly other structs in other places) there's no reason to continue execution.

I'd prefer to have a nice failed assertion failure output rather than the whole test panic.

Reply:
Absolutely, thanks for pointing this out!